test/test/InterfaceEntryPointTest.scala

```diff
@@ -18,7 +18,7 @@ import scala.collection.mutable.ListBuffer
  */
 class InterfaceEntryPointTest {
   @Test def runCompilerFromInterface = {
-    val sources = List("./tests/pos/HelloWorld.scala")
+    val sources = List("./tests/pos/HelloWorld.scala").map(p => new java.io.File(p).getPath())
```

Comment:
I think it would be good to factor out this incantation as `normalizePath(path: String)`, since it is used 3 times (currently).